chia/cmds/configure.py

```diff
@@ -237,6 +255,12 @@ def configure(
 )
 @click.option("--set-node-introducer", help="Set the introducer for node - IP:Port", type=str)
 @click.option("--set-farmer-peer", help="Set the farmer peer for harvester - IP:Port", type=str)
+@click.option("--set-solver-peer", help="Set the solver peer for farmer - IP:Port", type=str)
+@click.option(
+    "--set-solver-trusted-peers-only",
+    help="Enable/disable trusted peer requirement for solver connections",
```

Comment:
does this mean the solver only accepts connections from trusted farmers? or does it mean farmers only make connections to trusted solvers? I think this help message should clarify this.

I can see an argument for the trust needing to go both way. A malicious farmer can DoS a solver with tons of work. A malicious solver may steal the proof-of-work and farm its own blocks.

Reply:
the farmer only connects to solver that you explicitly input if its not local so thats redundant, this is for the solver accepting connections, i thoght that was kind of obvious but ill change the message